.github/workflows/ci-sdxl.yaml

```diff
@@ -99,4 +99,4 @@ jobs:
       working-directory: ${{ env.LIBSHORTFIN_DIR }}
       run: |
         ctest --timeout 30 --output-on-failure --test-dir build
-        pytest tests/apps/sd/e2e_test.py -v -s --system=amdgpu
+        HIP_VISIBLE_DEVICES=0 pytest tests/apps/sd/e2e_test.py -v -s --system=amdgpu
```

Comment:
> This sets HIP_VISIBLE_DEVICES from the .yaml -- it can be exposed as a pytest option as well (it's an option for the server CLI) but this is easiest and most obvious. #342 describes why this method was chosen for the runner (it's a little faster to init)

I think @saienduri also did something with `HIP_VISIBLE_DEVICES` for having multiple github actions runner instances on the same node, with one runner per GPU.

Reply:
This doesn't seem to be restricting the visible devices otherwise -- test failed due to a known multi-gpu bug.

Reply:
I think it's better to move this inside anyway, if anything, to test the server CLI option.

Reply:
@monorimet can you try using ROCR_VISIBLE_DEVICES. That's what we've been using in IREE and SHARK-TestSuite